CHANGELOG.md

```diff
@@ -37,7 +36,7 @@
   - Removed `AZ_CONTINUE`.
 - `az_storage_blobs_blob_client`:
   - `retry` field renamed to `retry_options` in `az_storage_blobs_blob_client_options`.
-  - Moved `az_context* context` parameter from `az_storage_blobs_blob_upload` into a public field on `az_storage_blobs_blob_upload_options`.
+  - Moved `az_context* context` parameter from `az_storage_blobs_blob_upload()` into a public field on `az_storage_blobs_blob_upload_options`.
```

Comment:
I was never part of a conversation about this change. Why was it done?

Reply:
It was done here: https://github.com/Azure/azure-sdk-for-c/pull/1005